config/default.json

```diff
@@ -29,7 +29,8 @@
     "enableSeeders": false
   },
   "awardsForAll": {
-    "enableExpiration": true
+    "enableExpiration": true,
+    "enableSessionExpiryWarning": true
```

Comment:
This applies to the whole website / all forms right? If so, I'd put this flag under the common `features` config rather than scoping to awards for all.